libraries/chain/db_block.cpp

```diff
@@ -560,8 +560,27 @@ processed_transaction database::_apply_transaction(const signed_transaction& trx
 { try {
    uint32_t skip = get_node_properties().skip_flags;
 
-   if( true || !(skip&skip_validate) )   /* issue #505 explains why this skip_flag is disabled */
-      trx.validate();
```

Comment:
I think here is not the only place that calls `trx.validate()`. There should be (much) more. Usually we don't change like this since it may mess up many things. In principle, validation should be always done regardless of db state. As I said somewhere else, we check db state related things in evaluators.

Reply:
By the way, we don't put individual operation related code in `db_block.cpp`. Again, the correct place to deal with operations is evaluator(s).